enterprise/check-spelling/custom.md

```diff
@@ -21,6 +21,23 @@ ephox {
 }
 ```
 
+### Dynamic Custom Dictionaries
+
+{{site.requires_jsspelling_war_2_110_0v}}
+
+Adding the `ephox.spelling.dynamic-custom-dictionaries` element and setting it to `true` means that the spelling service will periodically watch the `custom-dictionaries-path` for changes, and update the custom dictionaries accordingly. This prevents the need for a restart to pick up new changes to the custom dictionaries. The default value is `false`.
```

Comment:
Do we want to discuss how periodically, or maybe how to tell from looking at a response whether it's using the new dictionaries or not?

Reply:
Because it's not configurable, my preference was to keep it an implementation detail for now. It should be configurable, but making it configurable was postponed.

For the second part of your question, do you mean provide some sample cURL requests?

Reply:
Only include information the users need. Anything else is noise.